tests/test_dtel.py

```diff
@@ -0,0 +1,289 @@
+from swsscommon import swsscommon
+import time
+import re
+import json
+
+class TestDtel(object):
```

Comment:
can you include a test case that test port breakout case?

Reply:
This would require some time to get port breakout tested on VS platform. I did not see an existing VS test for port breakout. Can we add this test after upstreaming the current code?

Reply:
ok